pkg/activator/net/helpers_test.go

```diff
@@ -168,3 +169,54 @@ func TestGetServicePort(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkHealthyAddresses(b *testing.B) {
+	for _, n := range []int{1, 10, 100, 1000, 10000} {
+		b.Run(fmt.Sprint("addresses-", n), func(b *testing.B) {
+			ep := eps(10, n)
+			for i := 0; i < b.N; i++ {
+				healthyAddresses(ep, networking.ServicePortNameHTTP1)
+			}
+		})
+	}
+}
+
+func BenchmarkEndpointsToDests(b *testing.B) {
+	for _, n := range []int{1, 10, 100, 1000, 10000} {
+		b.Run(fmt.Sprint("addresses-", n), func(b *testing.B) {
+			ep := eps(10, n)
```

Comment:
just out of interest, is there a particular reason to pick `10`? Is that the default num activators somewhere?

Reply:
I took a high'ish number for activators there, but totally out of the blue tbh.

Reply:
heh, okie

Reply:
We can have it as a dimension